bazel/cloudbuild.yaml

```diff
@@ -6,10 +6,12 @@ steps:
 - name: 'gcr.io/cloud-builders/docker'
   args: ['build', '--tag=gcr.io/$PROJECT_ID/bazel', '.']
 - name: 'gcr.io/$PROJECT_ID/bazel'
+  env: ['BUILD_ID=$BUILD_ID']
```

Comment:
I'm not super pumped that bazel users have to remember to pass this through to get Bazel to pick it up, but GCB's model currently doesn't support this by default either.

We've gotten FRs to always passthrough envs like `$BUILD_ID`, `$COMMIT_SHA`, etc., as env vars, maybe this is just more evidence we should consider it.

Reply:
For a general FR, I would say: If you passthrough $PROJECT_ID and $BUILD_ID, then steps can call GetBuild to fetch the rest everything else.